crates/api_models/src/user/dashboard_metadata.rs

```diff
@@ -104,6 +105,12 @@ pub struct ProdIntent {
     pub is_completed: bool,
 }
 
+#[derive(Debug, serde::Deserialize, serde::Serialize, Clone)]
+pub struct ReconStatus {
```

Comment:
Nit: This _can_ also derive `Copy` for now (because all fields are bools).

Optional change, by the way.